src/Api.php

```diff
@@ -84,8 +84,8 @@ public static function getSecret()
         return self::$secret;
     }
 
-    public static function getFullUrl($relativeUrl)
+    public static function getFullUrl($relativeUrl,$apiVersion)
```

Comment:
can we set a default value here also for apiVersion?

Reply:
Done